pkgs/tools/security/eid-mw/default.nix

```diff
@@ -8,29 +8,24 @@
 
 stdenv.mkDerivation rec {
   pname = "eid-mw";
-  version = "4.4.16";
+  version = "4.4.27";
 
-  src = fetchFromGitHub {
-    sha256 = "1q82fw63xzrnrgh1wyh457hal6vfdl6swqfq7l6kviywiwlzx7kd"; 
-    rev = "v${version}";
-    repo = "eid-mw";
-    owner = "Fedict";
+  src = fetchurl {
+    url = "https://dist.eid.belgium.be/continuous/sources/eid-mw-${version}-v${version}.tar.gz";
+    sha256 = "02dbwgnkzpxkc4avs7zcrrwbjn2v59g5ffpb6vcfi80ybimw9hm6";
```

Comment:
Why is this moving away from GitHub, since it seems to have this version?

https://github.com/Fedict/eid-mw/releases/tag/v4.4.27

Also, it already seems to have a newer release?

https://github.com/Fedict/eid-mw/releases/tag/v5.0.5

Reply:
`dist.eid.belgium.be/` is the official release channel, where the releases are also signed. It seemed more appropriate to me. The 5.x releases seem to be unstable releases: they aren't mentioned on https://eid.belgium.be/en. They also failed to build. The AUR also has the same version: https://aur.archlinux.org/packages/eid-mw/ (they also use the official release channel there).

Reply:
> `dist.eid.belgium.be/` is the official release channel, where the releases are also signed. It seemed more appropriate to me.

I see, thanks! For now it would probably be be better to continue using GitHub as an upstream until something like #43233 lands. Someone might forget to GPG-verify the source tarball on version bumps. Until then GitHub gives better accountability (easy to get diffs). For 4.4.27 it doesn't matter anyway, because the git tag is signed by one of the developers.

> The 5.x releases seem to be unstable releases: they aren't mentioned on https://eid.belgium.be/en and in https://dist.eid.belgium.be/continuous/sources they have a commit hash in the released tarballs.

Ok, so then we should use 4.4.27.

Reply:
Sure, I'll edit it later tonight.